lib/new_relic/language_support.rb

```diff
@@ -88,7 +88,12 @@ def snakeize(string)
     end
 
     def bundled_gem?(gem_name)
-      defined?(Bundler) && Bundler.rubygems.all_specs.map(&:name).include?(gem_name)
+      defined?(Bundler) &&
```

Comment:
This condition feels a little complex with `defined?(Bundler) &&` at the top. What if you added a guard clause to check for Bundler instead?

Maybe something like: 
```
return false unless defined?(Bundler)

if ...... (rest of the code as-is)
``

Reply:
Less complex is where its at. https://github.com/newrelic/newrelic-ruby-agent/pull/2823/commits/b33eac4858938c8122354c9d769e6ab2c451cc61